lamindb_setup/dev/upath.py

```diff
@@ -410,16 +410,15 @@ def create_path(pathlike: Union[str, Path, UPath]) -> UPath:
         path = UPath(str(pathlike))  # UPath applied on Path gives Path back
     else:
         raise ValueError("pathlike should be of type Union[str, Path, UPath]")
-
     # remove trailing slash
     if path._parts[-1] == "":
         path._parts = path._parts[:-1]
 
-    if isinstance(path, S3Path):
-        path = UPath(path, cache_regions=True)
```

Comment:
It is important to keep cache_regions, otherwise s3fs doesn't work with newly created instances.

Reply:
Yes, I've just temporarily commented out.